src/components/label/label.e2e.ts

```diff
@@ -174,4 +175,39 @@ describe("calcite-label", () => {
 
     expect(eventDetail).toBeTruthy();
   });
+
+  describe("label click behavior", () => {
+    it("works when label is rendered before input", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html` <calcite-label></calcite-label> `);
+      await page.$eval(
+        "calcite-label",
+        (node) => ((node as HTMLElement).innerHTML = `<calcite-switch></calcite-switch>`)
+      );
+      await page.waitForTimeout(100);
+      await page.click("calcite-label", { clickCount: 1 });
+    });
+
+    it("works when label is rendered after input", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html` <calcite-switch id="switch"></calcite-switch> `);
+      await page.waitForTimeout(100);
+      await page.evaluate(() => `<calcite-label for='switch'></calcite-label>`);
+    });
+
+    it("works when label is removed and added back", async () => {
+      const page = await newE2EPage();
+      await page.setContent(
+        html`
+          <calcite-label for="switch">
+            <calcite-switch id="switch"></calcite-switch>
```

Comment:
Should the label already be in the page.setContent? Seems odd to be here and also created below.